pkl-executor/src/test/kotlin/org/pkl/executor/EmbeddedExecutorTest.kt

```diff
@@ -98,18 +98,15 @@ class EmbeddedExecutorTest {
         if (executor.isInitialized()) executor.value.close()
       }
     }
-    
+
     // a Pkl distribution that supports ExecutorSpiOptions up to v1
     private val pklDistribution1: Path by lazy {
-      val path = System.getProperty("pklDistribution025")?.toPath() ?:
```

Comment:
I don't understand why you removed this system property but not the other. And in the build script you still set both of them.

Reply:
Ah, thanks, should get rid of this too.